Cargo.toml

```diff
@@ -11,3 +17,11 @@ p256 = { version = "0.13.2", default-features = false, features = ["ecdsa"] }
 serde = { version = "1.0.197", default-features = false, features = ["std", "derive"] }
 serde_json = { version = "1.0.115", default-features = false, features = ["std"] }
 thiserror = { version = "1.0.58", default-features = false }
+bytes = { version = "1.6.0", default-features = false }
+prost = { version = "0.12.6", default-features = false }
+prost-build = { version = "0.12.6", default-features = false, optional = true }
+reqwest = { version = "0.12.4", default-features = false, features = ["json"] }
+dotenv = "0.15.0"
```

Comment:
https://github.com/allan2/dotenvy

Reply:
Ah yes. Thank you for pointing this out!